app/src/main/java/com/training/simpleloginform/MainActivity.java

```diff
@@ -43,16 +37,24 @@ protected void onCreate(Bundle savedInstanceState) {
     public void onClick(View view) {
         switch (view.getId()) {
             case R.id.submit_button:
-                String username = usernameEditText.getText().toString();
-                String password = passwordEditText.getText().toString();
+                Intent i = new Intent(this, SecondActivity.class);
```

Comment:
Let us have meaningful names for variables. Can we change 'i' to something meaningful like 'intent' ?